### PR TITLE
Xyc/function local commands

### PR DESCRIPTION
**Pull request by xyc**
### What does this PR do?
Commands for running a Salesforce function locally. Will have another PR on starting / stopping function.

### What issues does this PR fix or reference?
@W-7958829@

### CODE REVIEW

**Review by jag-sfdc — APPROVED** [review]

File: packages/salesforcedx-vscode-core/package.nls.json [hunk 1]
```diff
@@ -66,5 +66,8 @@
   "experimental_deploy_retrieve_description": "Enable performance enhancements to the deploy/retrieve experience.",
   "experimental_apex_library_description": "Use Apex Library for Apex CLI commands",
   "force_function_create": "SFDX: Create Function",
+  "force_function_start": "SFDX: Start Function",
```
Comment (xyc) [inline packages/salesforcedx-vscode-core/package.nls.json:69]: @ekapner Command names for local functions. Please let me know if this looks good. This won't be exposed until we flip the flag.

Comment (ekapner) [reply packages/salesforcedx-vscode-core/package.nls.json:69]: @danyusf - These looks good to me; and follow the command names and verbiage you use in the Dev Guide.

**Review by ekapner — COMMENTED** [review]
The commands look good to me. Tagged Dan Yu in case he has any feedback.

File: packages/salesforcedx-vscode-core/package.nls.json (hunk 1 above)
Comment (danyusf) [reply packages/salesforcedx-vscode-core/package.nls.json:69]: LGTM (assuming this basically wraps the evergreen:function:start cmd)

Comment (xyc) [reply packages/salesforcedx-vscode-core/package.nls.json:69]: Thanks, yeah it's a wrapper over the plugin command

**Pull request merged**